adapters/rest-adapter-core/src/main/java/uk/gov/justice/services/adapter/rest/JsonSchemaValidationInterceptor.java

```diff
@@ -38,13 +43,15 @@ public Object aroundReadFrom(final ReaderInterceptorContext context) throws IOEx
         if (context.getMediaType().getSubtype().endsWith(JSON_MEDIA_TYPE_FORMAT)) {
             final String charset = extractCharset(context);
             final String payload = IOUtils.toString(context.getInputStream(), charset);
-
+            final String name = extractName(context);
```

Comment:
extractName can go inline again, because we don't use it in the exception handling